quranHome/src/commonMain/kotlin/com/islamversity/quran_home/feature/home/QuranHomeProcessor.kt

```diff
@@ -28,4 +28,13 @@ class QuranHomeProcessor(
             }
             .navigateTo(navigator)
     }
+
+    private val settingsClicked: FlowBlock<QuranHomeIntent, QuranHomeResult> = {
+        ofType<QuranHomeIntent.SettingsClicked>()
+            .map {
+                //TODO ADD TRANSITION
```

Comment:
what is a transaction?
do we really need it?

Reply:
No